### PR TITLE
Add log configuration in coredns docs

### PR DESCRIPTION
**Pull request by pipo02mix**
Add log plugin configuration in coredns docs page

### CODE REVIEW

File: src/content/guides/advanced-coredns-configuration/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ __Warning:__ Please do not edit any of the other CoreDNS related resources. Only
 
 ## Cache settings
 
-By default we set the cache TTL for CoreDNS to 30 seconds. You can customize the cache settings of CoreDNS by setting the value of the cache field in the user ConfigMap like following.
+By default we set the cache TTL for CoreDNS to 30 seconds. You can customize the cache settings of CoreDNS by setting the value of the cache field in the user ConfigMap like following:
```
Comment (rossf7) [inline src/content/guides/advanced-coredns-configuration/index.md]:
```suggestion
By default we set the cache TTL for CoreDNS to 30 seconds. You can customize the cache settings of CoreDNS by setting the value of the cache field in the user ConfigMap like this:
```

Minor but flows better for me.

File: src/content/guides/advanced-coredns-configuration/index.md [hunk 2]
```diff
@@ -38,6 +38,18 @@ Above setting increases the TTL to 60 seconds.
 
 The cache plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/cache/).
 
+## Logs
+
+By default, we set the log level for CoreDNS to `denial` and `error`. You can tun these settings by adding a property `log` in the user ConfigMap like following:
```
Comment (rossf7) [inline src/content/guides/advanced-coredns-configuration/index.md]:
```suggestion
By default, we set the log level for CoreDNS to `denial` and `error`. You can tun these settings by adding a property `log` in the user ConfigMap like this:
```

File: src/content/guides/advanced-coredns-configuration/index.md [hunk 3]
```diff
@@ -38,6 +38,18 @@ Above setting increases the TTL to 60 seconds.
...
+
+```yaml
+data:
+  log: 
+  - all
+```
+
+For knowing the exact details of each log level log plugin, please read the [upstream documentation](https://coredns.io/plugins/log/).
```
Comment (rossf7) [inline src/content/guides/advanced-coredns-configuration/index.md]:
```suggestion
To know the exact details of each log level log plugin, please read the [upstream documentation](https://coredns.io/plugins/log/).
```

**Comment by pipo02mix** [comment]
Damn it, I did not apply suggestions. Doing it in following PR

**Pull request merged**